h/auth/__init__.py

```diff
@@ -10,15 +10,16 @@
     AuthenticationPolicy,
     TokenAuthenticationPolicy,
 )
-from h.auth.util import default_authority, groupfinder
+from h.auth.util import default_authority
+from h.security import principals_for_userid
 from h.security.encryption import derive_key
 
 __all__ = ("TOKEN_POLICY",)
 
 log = logging.getLogger(__name__)
 
 # We export this for the websocket to use as it's main policy
-TOKEN_POLICY = TokenAuthenticationPolicy(callback=groupfinder)
+TOKEN_POLICY = TokenAuthenticationPolicy(callback=principals_for_userid)
```

Comment:
`groupfinder` is an absolutely horrible name. This is because pyramid refers to principals as "groups" in certain contexts, but it's just mad confusing for us.